### PR TITLE
 [FLINK-36633] Add support for Flink 1.20 in Flink OpenSearch connector

### PR DESCRIPTION
**Pull request by reta**
Update to support Flink 1.20.0

### CODE REVIEW

**Comment by reta** [comment]
@snuyanzin could you please help me with this one? thanks!

**Comment by snuyanzin** [comment]
yep, sure will look at it

can you please create a dedicated jira issue for this activity and use it instead of just `hotfix`
like e.g. https://issues.apache.org/jira/browse/FLINK-35109

File: pom.xml [hunk 1]
```diff
@@ -370,18 +370,11 @@ under the License.
...
-				<artifactId>byte-buddy</artifactId>
-				<version>1.14.10</version>
-			</dependency>
-
 			<dependency>
 				<!-- mockito/powermock mismatch -->
 				<groupId>net.bytebuddy</groupId>
 				<artifactId>byte-buddy-agent</artifactId>
-				<version>1.14.10</version>
+				<version>1.14.13</version>
```
Comment (snuyanzin) [inline pom.xml:377]:
```suggestion
				<version>1.14.19</version>
```
nit

**Comment by snuyanzin** [comment]
Thanks, for the contribution @reta
looks good to me, 
i have one minor comment

File: pom.xml (hunk 1 above)
Comment (reta) [reply pom.xml]: Updated, thanks @snuyanzin !

**Review by snuyanzin — APPROVED** [review]

**Pull request merged**